src/Coduo/PHPMatcher/Matcher/Pattern/RegexConverter.php

```diff
@@ -19,6 +19,8 @@ public function toRegex(TypePattern $type)
                 return "(\\-?[0-9]*)";
             case 'double':
                 return "(\\-?[0-9]*[\\.|\\,][0-9]*)";
+            case 'null':
+                return "(null|NULL)";
```

Comment:
I guess it have no sense to match "lorem ipsum null" against pattern "lorem ipsum @null@" in texts, maybe instead of supporting @null@ type matcher we could handle UnknownTypeException in TextMatcher and fail matching when this exception is throwed with error message that "@null@ type matcher is not supported in text matching".
What do you think @rklak ?